Makefile

```diff
@@ -23,7 +24,7 @@ install: certs
 	#
 	#=============================================
 	#
-	helm install connaisseur helm --atomic
+	helm install connaisseur helm --atomic --set alerting.cluster=$(CLUSTER)
```

Comment:
@phbelitz I remember you wanted to have it possible to install Connaisseur via `make` as well as `helm`. I added a custom failure for `helm` if it is invoked without this cluster variable. Is that sufficient for you?

Reply:
looks really good! we might use the failure at other parts as well.